src/styles/styles.js

```diff
@@ -25,6 +25,7 @@ import CONST from '../CONST';
 import cursor from './utilities/cursor';
 import userSelect from './utilities/userSelect';
 import textUnderline from './utilities/textUnderline';
+import Colors from './colors';
```

Comment:
I don't think we use Colors here. We use Colors in themeColors and themeColoros in styles.